.github/workflows/ci_dockers.yml

```
@@ -97,7 +97,7 @@ jobs:
       matrix:
         # the config used in '.github/workflows/ci_test-conda.yml'
         python_version: ["3.8"]
-        pytorch_version: ["1.7", "1.8", "1.9", "1.10"]
+        pytorch_version: ["1.8", "1.9", "1.10"]
```

```suggestion
        pytorch_version: ["1.7", "1.8", "1.9", "1.10"]
```

Rolling back some commits to see what went wrong. https://github.com/PyTorchLightning/pytorch-lightning/runs/5411681687?check_suite_focus=true failing for 1.8, 1.9